### ISSUE TITLE
Improve UI for touch screen devices

### ISSUE THREAD

**Issue by jaw111** [dvcama/LodLive#23]
When I touch and drag a circle on a touch-screen device, the whole screen scrolls rather than repositioning the selected circle.

This makes it impossible to re-arrange entities using a tablet/phone as one would intuitively expect.

**Comment by dvcama** [comment]
yes that would be nice... 3 years later and I still have no time to do it
sorry
